src/tools/storybook-addon-system-switcher/src/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
I am not sure whether there are official conventions regarding the index file of an addon, however in most of the addons I've seen, the `index.js` is a place for making use of `makeDecorator`, i.e. exporting a decorator interface.

If it's only a react component, it could be as single as `Switcher.jsx` and no issues with the linter.

Reply:
It's sometimes useful to have an empty `index.js` with `// noop` in oder to avoid breaking module systems but instructing a no-operation approach.